src/network.rs

```diff
@@ -413,7 +412,21 @@ impl EventLoop {
 							.add_address(&peer_id, multiaddr);
 					}
 				},
-				_ => (),
+				MdnsEvent::Expired(addrs_list) => {
+					for (peer_id, multiaddr) in addrs_list {
+						debug!(
```

Comment:
Its also possible to write: `debug!("MDNS got expired peer with ID: {peer_id:#?} and Address: {multiaddr:#?}");`